### PR TITLE
Milestone 1 App Setup

### PR DESCRIPTION
**Pull request by macbright**
Description

this milestone is the first milestone for the project. in this the milestone, i setup the required technologies and development environment needed to build the app.

### CODE REVIEW

**Comment by macbright** [comment]
i will fix the linter error later. can't fix it now cos i finished the whole project before requesting for the first review

File: .stickler.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+    flake8:
+    eslint:
+        config: './eslint.config'
+  
+
+files:
+    ignore:
+        - 'src/generated/*'
+        - 'webroot/bower_components/*'
+        - 'js/index.js'
```
Comment (bolah2009) [inline .stickler.yml]: Let's use the content here instead: https://github.com/microverseinc/linters-config/blob/master/react-redux/.stickler.yml

File: eslint.config [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  "extends": "airbnb-base",
+    rules: {
+    'no-shadow': 0,
+    'no-param-reassign': 0,
+    'eol-last': 0,
+    "indent": ["error", "tab", { "SwitchCase": 1 }],
+    'no-tabs': 0,
+    'skipBlankLines': true 
+  }
+}
```
Comment (bolah2009) [inline eslint.config]: Consider using this file and it's content instead: https://github.com/microverseinc/linters-config/blob/master/react-redux/.eslintrc.json

**Review by bolah2009 — CHANGES_REQUESTED** [review]
Hi @macbright, nice job with the project setup.

Please let's use the lint configuration in this repo: https://github.com/microverseinc/linters-config/tree/master/react-redux also add a descriptive PR name and description, Note that you can edit your current PR name and description (See below). 

### Rename PR
  rename PR
-----
### Update PR description
  update-PR-description-html


#### :recycle:  Make these changes and submit another code review request. :+1: 

#### Feel free to reach me on slack if you having issues implementing these features. :+1: 

Good luck with your next submission. :star2: 

Cheers
[Bola](https://bolabuari.com/)

###### PS: Note that comments tagged as `OPTIONAL` are not part of the minimum requirements to approve this PR

**Review by bolah2009 — APPROVED** [review]
It's a 👍🏾 from me!

### :heavy_check_mark: Approved!

Good luck with your next milestones. :star2: 

Cheers
[Bola](https://bolabuari.com/)

**Pull request merged**